mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -202,10 +202,14 @@ func resourceBigtableGCPolicyUpsert(d *schema.ResourceData, meta interface{}) er
 	tableName := d.Get("table").(string)
 	columnFamily := d.Get("column_family").(string)
 
-	err = retryTimeDuration(func() error {
+	retryFunc := func() (interface{}, error) {
 		reqErr := c.SetGCPolicy(ctx, tableName, columnFamily, gcPolicy)
-		return reqErr
-	}, d.Timeout(schema.TimeoutCreate), isBigTableRetryableError)
+		return "", reqErr
+	}
+	// The default create timeout is 20 minutes.
+	timeout := d.Timeout(schema.TimeoutCreate)
+	pollInterval := time.Duration(30) * time.Second
+	_, err = retryWithPolling(retryFunc, timeout, pollInterval, isBigTableRetryableError)
```

Comment:
```suggestion
	// Mutations to gc policies can only happen one-at-a-time and take some amount of time.
	// Use a fixed polling rate of 30s based on the RetryInfo returned by the server rather than
	// the standard up-to-10s exponential backoff for those operations.
	_, err = retryWithPolling(retryFunc, timeout, pollInterval, isBigTableRetryableError)
```